src/main/resources/com/google/api/codegen/py/init.snip

```diff
@@ -0,0 +1,79 @@
+@extends "py/common.snip"
+
```

Comment:
I would call this file `sample_init.snip`, since it is only used for samples. This will reduce confusion when looking at the template files in the directory, trying to figure out which one is used when.



[ Down the line, it may be worth moving the sample files into
* `.../codegen/py/samples/inline` for the inline samples
* `.../codegen/py/samples/standalone` for the standalone samples
* `.../codegen/py/samples/` for the files, like this one, common to all samples.

We can do this later. ]

Reply:
Done